src/pages/MovieDetail/DetailOption.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+const DetailOption = ({ movieData }) => {
+  const { price } = movieData;
+  const [count, setCount] = useState(1);
+  const navigate = useNavigate();
+
+  const onCart = () => {
+    fetch('http://10.58.52.88:3000/myShallWe/pick', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        authorization:
+          'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjEsImlhdCI6MTY3NTU4NDcyNX0.tylNeMdSFT6iAxe6kJyfuoDpKduEoik4kTYM_PfiW1c',
+      },
+      body: JSON.stringify({
+        id: 1,
+        counts: count,
+        price: price * count,
+      }),
+    }).then((response) => {
+      if (response.ok === true) {
+        if (window.confirm('마이페이지에 담았습니다. 마이페이지로 이동하시겠습니까?')) {
+          navigate('/');
+        }
+        return;
+      }
```

Comment:
```suggestion
      if (!response.ok) return;
      if (window.confirm('마이페이지에 담았습니다. 마이페이지로 이동하시겠습니까?')) {
        navigate('/');
      }
```

이런 식으로 바꿔줘도 괜찮지 않을까요?